worlds/witness/docs/setup_en.md

```diff
@@ -43,4 +43,4 @@ The Witness has a fully functional map tracker that supports auto-tracking.
 3. Click on the "AP" symbol at the top.
 4. Enter the AP address, slot name and password. 
 
-The rest should take care of itself! Items and checks will be marked automatically, and it even knows your settings - It will hide checks & adjust logic accordingly.
+The rest should take care of itself! Items and checks will be marked automatically, and it even knows your options - It will hide checks & adjust logic accordingly.
```

Comment:
Ugh. Yeah, another instance where I'm really frustrated by the ambiguity of the word "option". But this one is more acceptable. I would personally probably also reword this to say something like "the options you picked", but it's not as necessary imo

Reply:
```suggestion
The rest should take care of itself! Items and checks will be marked automatically, and it even knows the options you chose - It will hide checks & adjust logic accordingly.
```

Reply:
Yeah I like that :3